wodles/aws/aws_s3.py

```diff
@@ -17,13 +17,19 @@
 #   7 - Unexpected error querying/working with objects in S3
 #   8 - Failed to decompress file
 #   9 - Failed to parse file
+#   10 - pyarrow module missing
```

Comment:
This is a very weird exception. We have other hard dependencies for the module and we are not mentioning them explicitly.

Reply:
We followed the same pattern as the one used for the `boto3` package. 
```
#   4 - boto3 module missing
```
Since this dependency is crucial for the integration to work, handle the `parquet` files fetched, and it will be part of the `AWS pip dependencies` section of the documentation, we decided it should have its own error code.